hikari/colors.py

```diff
@@ -517,12 +517,12 @@ def of(cls, value: Colorish, /) -> Color:
             if len(value) != 3:
                 raise ValueError(f"Color must be an RGB triplet if set to a {type(value).__name__} type")
 
-            if any(isinstance(c, float) for c in value):
-                r, g, b = value
+            r, g, b = value
+
+            if isinstance(r, float) and isinstance(g, float) and isinstance(b, float):
```

Comment:
For future reference: This has changed from being `any` to being the equivalent of `all`. This was never documented to work with a mixture of floats and ints, and even the examples dont show that behaviour, so technically just impl behaviour